src/components/linear-progress/linear-progress.tsx

```diff
@@ -15,6 +18,13 @@ const PERCENT = 100;
     styleUrl: 'linear-progress.scss',
 })
 export class LinearProgress {
+    /**
+     * Defines the language for translations.
+     * Will translate the translatable strings on the components.
+     */
+    @Prop({ reflect: true })
+    public language: Languages = 'en';
```

Comment:
It feels like we ought to default to the language requested by the browser here, if it's one we support, and otherwise use English. But I guess this PR is big enough without that addition.

Reply:
Yeah, true 👍